internal/applicationsnapshot/report.go

```diff
@@ -94,20 +97,35 @@ type TestReport struct {
 
 // Possible formats the report can be written as.
 const (
-	SummaryMarkdown = "summary-markdown"
 	JSON            = "json"
 	YAML            = "yaml"
-	APPSTUDIO       = "appstudio"
+	Text            = "text"
```

Comment:
As you're refactoring, we could define a `type OutputFormatType string` and have these typed like that...

Reply:
Yeah I was thinking about that. Maybe in another commit since I merged #1640 already.